shared/domains/chelonia/utils.js

```diff
@@ -498,12 +485,13 @@ export const recreateEvent = async (entry: GIMessage, rootState: Object): Promis
 
   if (!newRawOpV) return
 
+  const newOp = opT === GIMessage.OP_ATOMIC && (newRawOpV: any).length === 1
+    ? (newRawOpV: any)[0]
+    : [opT, newRawOpV]
```

Comment:
What's going on here? Are you sure this is right? Is `? (newRawOpV: any)[0]` correct?

Why is this being done? Is it rewriting what the user is sending? We should probably keep the developer intent...

Reply:
Reverted